interpreter/interpreter.go

```diff
@@ -175,7 +177,77 @@ func (itrp *Interpreter) doStatement(stmt semantic.Statement, scope values.Scope
 }
 
 func (itrp *Interpreter) doOptionStatement(s *semantic.OptionStatement, scope values.Scope) (values.Value, error) {
-	return itrp.doAssignment(s.Assignment, scope)
+	switch a := s.Assignment.(type) {
+	case *semantic.NativeVariableAssignment:
+		init, err := itrp.doExpression(a.Init, scope)
+		if err != nil {
+			return nil, err
+		}
+		// Use an empty string as the package name as we don't know its name.
+		// This will have one of two behaviors:
+		//     1. The option key will be found in the prelude and applied there.
+		//     2. The option key will not be found in the prelude and the
+		//        interpreter will handle adding the new option to the current package.
+		return itrp.setOption(scope, "", a.Identifier.Name, init)
```

Comment:
The more I think about it, I'm not sure I like this. That is, conflating update and extend semantics. If there weren't any options defined in the prelude, then we wouldn't need this behavior as option updates would only be specified via member assignments. This could be achieved by defining `now` in the `system` package. Thoughts? Is having to specify `system.now()` a bad user experience?